src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/deprecated_parameter.txt

```diff
@@ -0,0 +1,6 @@
+ ------------------------------------------------------------------------------------- ------- 
+ [32m Parameter                                                                           [39m [32m Value [39m 
+ ------------------------------------------------------------------------------------- ------- 
+  deprecated_foo                                                                        bar    
+  [33m(Since symfony/framework-bundle 6.4: The parameter "deprecated_foo" is deprecated.)[39m          
```

Comment:
can you check the rowspan?

Reply:
Yes :), look at that:
<img width="1065" alt="Screenshot 2023-09-24 at 5 26 59 PM" src="https://github.com/symfony/symfony/assets/10107633/5fdc5582-cd06-4967-88b1-57daf6f201f9">